verilog/analysis/checkers/always_comb_blocking_rule.h

```diff
@@ -56,7 +56,7 @@ class AlwaysCombBlockingRule : public verible::SyntaxTreeLintRule {
   const Matcher always_comb_matcher_ =
       NodekAlwaysStatement(AlwaysCombKeyword());
 
-  std::vector<verible::LintViolation> violations_;
+  std::set<verible::LintViolation> violations_;
```

Comment:
Replace `#include <vector>` with `#include <set>` in all of these rules' .h files.

Reply:
Replaced